src/Sulu/Bundle/PageBundle/Command/WebspaceCopyCommand.php

```diff
@@ -557,6 +565,43 @@ protected function updateTeaserSelection(
         }
     }
 
+    /**
+     * Updates references in structure for content type `link`.
+     *
+     * @param string $localeSource
+     * @param string $localeDestination
+     */
+    protected function updateLinkSelection(
+        array &$structureArray,
+        PropertyMetadata $property,
+        $localeSource,
+        $localeDestination
+    ) {
+        if (!isset($structureArray[$property->getName()]['provider'])) {
+            return;
+        }
+
+        if ('page' !== $structureArray[$property->getName()]['provider']) {
```

Comment:
The `page` not require a provider be set because of bc reasons old sulu-links didn't have that. And it will fallback to the default provider:

https://github.com/sulu/sulu/blob/cf4451ac29aaf328765ce3c43f0867c0e5b95aa7/src/Sulu/Bundle/MarkupBundle/Markup/LinkTag.php#L25

```suggestion
        if (!\is_array($structureArray[$property->getName()])) {
            return;
        }

        $linkProvider = $structureArray[$property->getName()]['provider'] ?? LinkTag::DEFAULT_PROVIDER;

        if ('page' !== $linkProvider) {
```